build/Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM registry.ci.openshift.org/stolostron/builder:go1.20-linux AS plugin-builder
-ENV POLICY_GENERATOR_TAG=release-2.9
+ENV POLICY_GENERATOR_TAG=v1.12.4
```

Comment:
@mikeshng I'll revert this, but for your awareness, this shouldn't have been changed in `stolostron`.

Reply:
...actually, this one is one us. We should have bumped it to `release-2.10` a while ago. I'll put that on our radar for next time.